target/linux/ramips/image/mt7620.mk

```diff
@@ -69,7 +78,9 @@ define Device/ArcherMR200
   $(Device/Archer)
   DTS := ArcherMR200
   SUPPORTED_DEVICES := mr200
-  TPLINK_BOARD_ID := ArcherMR200
+  TPLINK_FLASHLAYOUT := 8Mmtk
```

Comment:
Here is typo. MR200 use 8MLmtk flash layout.

Reply:
Good catch, fixed thanks!